src-plugins/manualRegistration/manualRegistrationLandmarkController.cpp

```diff
@@ -147,9 +147,15 @@ manualRegistrationLandmarkController::manualRegistrationLandmarkController()
 //----------------------------------------------------------------------------
 manualRegistrationLandmarkController::~manualRegistrationLandmarkController()
 {
+    delete Points_Moving;
```

Comment:
It would be safer to precede this with a reset to make sure all landmarks are destroyed.